docs/en/ingest-management/integrations/managed-integrations-content.asciidoc

```diff
@@ -0,0 +1,18 @@
+[[managed-integrations-content]]
+= Managed integrations content
+
+++++
+<titleabbrev>Managed integrations content</titleabbrev>
+++++
+
+Most integration content installed by {fleet} isn’t editable. This content is tagged with a **Managed** badge in the {kib} UI. Managed content itself cannot be edited or deleted, however managed visualizations, dashboards, and saved searches can be cloned.   
+
+[role="screenshot"]
+image::images/system-managed.png[An image of the new managed badge.]
+
+The cloned content is unlinked with the original reference configurations, so becomes independent from the original managed content. For example, with a cloned dashboard, the cloned panels become entirely self-contained copy without any dependencies on the original configuration. Clones can be customized and modified without accidentally affecting the original.
```

Comment:
> The cloned content is unlinked with the original reference configurations, so becomes independent from the original managed content.

I don't think we can call this a general rule. For example, a managed visualization may use the `logs-*` data view which is a managed data view. And the cloned version will, too.

The only case where the unlinking happens is with dashboard panels, as you've described here!

Reply:
Ah ok, sorry that was a bad assumption. Let me rework this and also update the corresponding Kibana page.